backend/app/services/llm.py

```
@@ -0,0 +1,46 @@
+def generate_reply(system: str, history: list[dict]) -> str:
+    """system プロンプト＋会話履歴から、面接官の次の発言を1つ生成して返す。
+
+    history: [{"role": "assistant" | "user", "content": "..."}, ...]
+    """
+    import anthropic
+    from anthropic import AnthropicBedrockMantle  # 遅延import
+
+    client = AnthropicBedrockMantle(aws_region=os.getenv("AWS_REGION", "ap-northeast-1"))
+
+    # Anthropic API は最初のメッセージが user である必要がある。
+    # 履歴が空（面接の1問目）はキックオフ用の user メッセージを立てる。
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)